dash-pipeline/tests/saithrift/ptf/thrift/test_thrift_session.py

```diff
@@ -1,7 +1,7 @@
 from sai_thrift.sai_headers import *
 from sai_base_test import *
 
-class TestSaiThriftSession(ThriftInterfaceDataPlane):
+class TestSaiThriftSession(SaiHelperSimplified):
     """ Test saithrift client connection only"""
     def setup(self):
```

Comment:
FYI: `setup()` and `teardown()` are never called in this class.
The method names should be `setUp()` and `tearDown()` - with camelCase.

Reply:
Fixed,thanks!